lib/datasource.js

```diff
@@ -348,6 +348,14 @@ DataSource.prototype.setup = function(name, settings) {
     if (settings.initialize) {
       connector = settings;
     } else if (settings.connector) {
+      // some connectors are aliases to loopback-component-storage
+      if (['object-storage'].indexOf(settings.connector) > -1) {
```

Comment:
I have mixed feelings about handling aliases inside juggler. I am concerned that such solution won't scale long term as more and more aliases are added. I am also worried that 3rd-party connectors maintained by our community cannot use this mechanism without waiting for us to review & approve such addition.